host/4/bullseye/amd64/node/node16/node16.Dockerfile

```diff
@@ -0,0 +1,72 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.0.0-preview.5.16714
+FROM mcr.microsoft.com/dotnet/sdk:6.0.100-rc.2 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 5) && \
+    BRANCH_NAME=release/$(echo ${HOST_VERSION} | cut -d'.' -f 1-4) && \
+    SUFFIX=$(echo ${HOST_VERSION} | cut -d'-' -f 2 | cut -d'.' -f 1-2) && \
+    git clone --branch $BRANCH_NAME https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:VersionSuffix=$SUFFIX /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
+    EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.8.1_linux-x64.zip && \
+    apt-get update && \
+    apt-get install -y gnupg wget unzip && \
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION/$EXTENSION_BUNDLE_FILENAME && \
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME && \
+    EXTENSION_BUNDLE_VERSION_V2=2.6.1 && \
```

Comment:
please update to latest extension bundles . See https://github.com/Azure/azure-functions-docker/commit/9e6ddfd52a39f554886a808e4cddb9c3771d0f93 for reference